src/components/canvas-instance.ts

```diff
@@ -95,7 +105,6 @@ export class CanvasInstance extends BaseComponent {
     this._$element = $(this.options.target);
     this._data = this.options.data;
     this.$playerElement = $('<div class="player player--loading"></div>');
-    this.$playerElement.hide();
```

Comment:
I had to remove this in order to see the controls container on the demo application. Doing so didn't seem to have a negative effect when building the library into UV either, so I'm not sure if this is needed.